source/administration/config-settings.rst

```diff
@@ -2130,6 +2139,9 @@ Enable Amazon S3 Debugging
 | This feature's ``config.json`` setting is ``"AmazonS3Trace": false`` with options ``true`` and ``false`` for above settings respectively.                            |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Test Connection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
There was no help text in the Test Connection button, what do I write here as a description?

Reply:
The button just makes sure we can access the server and the settings are valid